its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -232,14 +231,32 @@ public static String replaceLtsVersion(String version) {
     return version;
   }
 
-  public static void runNuGet(Orchestrator orch, Path projectDir, String... arguments) {
+  public static void runNuGet(Orchestrator orch, Path projectDir, Boolean useDefaultVSCodeMSBuild, String... arguments) {
+    Path nugetPath = getNuGetPath(orch);
+    int r = 1;
+    if (useDefaultVSCodeMSBuild)
```

Comment:
The two branches are very similar. You could extract the command that is created into a variable, than based on the boolean parameter add the additional argument or not and  after the if you could just execute it and assert the value.